content/md/administration/catalog-volume-monitoring_fr.md

```diff
@@ -0,0 +1,72 @@
+---
+id: catalog-volume-monitoring
+themes: administration, akeneo-concepts, catalog-settings 
+title: Controllr la **taille de votre catalogue** (since 2.3)
+popular: false
+related: what-is-a-product-value, what-is-an-attribute, what-is-a-family, what-is-a-channel, what-is-a-locale, what-is-a-category
+---
+
+# Un écran dédié
+
+Un écran est disponible dans le PIM afin de controller la taille du catalogue sur plusieurs axes.
+
+Cet écran contient par exemple le nombre de familles ou le nombre moyen d'attributs par famille.
+
+![Catalog volume monitoring dashboard](../img/Activity_CatalogVolumeMonitoring_fr.png)
+
+::: warning
+Cet écran n'est disponible que depuis la version 2.3 du PIM.
+:::
+
+# Où est-ce?
+
+Voici les étapes afin d'accéder à ce superbe (;)) écran:
+
+1. Cliquer sur le menu `Activité`
+2. Cliquer sur le sous-menu `Volume du catalogue`
+3. Et voilà !
+
+# Comment comprendre ce dashboard ?
+
+## Les différentes sections
+
+L'écran `Volume du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit
+
+La première valeur affichée dans le widget représente le nombre total de [valeurs de produit](/articles/what-is-a-product-value.html) contenues dans le PIM.
```

Comment:
Why did you remove the fact that this is the most important info?

Reply:
I don't like "L'information la plus importante" as a title. It's kind of weird. It seems as a "bait title" and it indicates nothing.
I prefer to add a sentence.


Also, is it really the most important information? Maybe it's the completeness of the catalog after all.

Reply:
This is the most important information of the screen yes. Completeness is also important but this is for another purpose.
It's ok if you don't put it as a title. But I think we should really emphasize somewhere on the fact this is the most representative number of your catalog size, so this is the axis we should always have a close look at.

Reply:
I will add it in the title

Reply:
done